src/resources/clinical-health-v0-template.json

```diff
@@ -0,0 +1,180 @@
+{
+  "baseUrl": "https://sandbox-api.va.gov/services/clinical-fhir/v0/r4",
+  "authentication": {
+    "type": "ccg",
+    "clientIdEnvVar": "CHAPI_CCG_CLIENT_ID",
+    "clientSecretEnvVar": "CHAPI_CCG_CLIENT_SECRET",
+    "audienceEnvVar": "CHAPI_CCG_AUDIENCE",
+    "oauthUrlEnvVar": "CHAPI_CCG_OAUTH_URL",
+    "scopes": "launch system/AllergyIntolerance.read system/Condition.read system/MedicationDispense.read system/MedicationRequest.read system/Observation.read system/Patient.read system/Practitioner.read",
+    "launchPatientEnvVar": "MAGIC_PATIENT_ICN"
+  },
+  "testPatients": [
+    "32000225",
+    "5000335",
+    "25000126",
+    "2000190",
+    "1012853802V084487",
+    "1011568236V299349",
+    "1012845331V153043"
+  ],
+  "patientFields": [
+    {
+      "name": "Name",
+      "path": ".name[0].text"
+    },
+    {
+      "name": "Birthdate",
+      "path": ".birthDate"
+    }
+  ],
+  "resources": [
+    {
+      "type": "AllergyIntolerance",
+      "staticFields": [
+        {
+          "name": "CodeSystem",
+          "value": "http://www.nlm.nih.gov/research/umls/rxnorm"
+        }
+      ],
+      "jqFields": [
```

Comment:
I was almost thinking something like:
```
columns: [
  {name: x, static: https://test.com },
  {name: y, jq: ".code"}
]
```

but I think that may complicate things a bit. I think you already have to process the file twice though, so maybe not.